lib/jsonwp-proxy/proxy.js

```diff
@@ -26,15 +26,15 @@ class JWProxy {
   }
 
   endpointRequiresSessionId (endpoint) {
-    return !_.includes(['/session', '/sessions', '/status'], endpoint);
+    return !_.includes(['/session', '/sessions', '/status', '/homescreen'], endpoint);
```

Comment:
Alright. It should be removed from this list, then, and just added as a normal endpoint.